tracks/javascript/exercises/two-fer/mentoring.md

```diff
@@ -89,33 +94,44 @@ This applies if a student has one of the following:
 - `expression ? consequent : alternate`
 - `literal || 'you'`
 
-In each case, they're handling the fact that sometimes a name is not passed in. Suggest they use a [default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)
-for the parameter (usually called `name`).
+In each case, they're handling the fact that sometimes a name is not passed in.
+Suggest they use a [default value][ref-default-parameter] for the parameter
+(usually called `name`).
 
 #### String concatenation
 
-This applies if a student does _not_ use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
+This applies if a student does _not_ use a [template literal][ref-template-literal]:
 
 - `'One for ' + name + ', one for me'`
 
-This is the perfect time to point them towards templated string literals and how to interpolate values.
+This is the perfect time to point them towards templated string literals and how
+to interpolate values.
 
 #### CommonJS exports
 
 This applies if the student does _not_ use ES6 style exports:
 
 - `module.exports = { twoFer }`
 
-Explain that we use babel, and this allows the student to use all new language features. They may use ES6 style `export`
-as well as any other ES6 feature. If this is the **only** comment, approve but give this suggestion for future exercises.
+Explain that we use babel, and this allows the student to use all new language
+features. They may use ES6 style `export` as well as any other ES6 feature. If
+this is the **only** comment, approve but give this suggestion for future
+exercises.
 
 ### Talking points
 
-- The difference of intent between a default value and using a falsy logical or (`name || 'you'`). What happens when the
-  input is `null`, what happens when the input is `''`, what happens when the input is ... `0`?
-- The difference between `===` and `==`, `!==` and `!=` ([equality comparisons and sameness](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)).
-  You should suggest they always use strict equality, but since the optimal solution doesn't have a conditional, it
-  does not apply to this exercise.
-- On exercism there is no need for [defensive programming](https://en.wikipedia.org/wiki/Defensive_programming). This
-  means they should not test if the input is a string, or something else unforeseen. The test cases should indicate what
-  a function might be called with.
+- The difference of intent between a default value and using a falsy logical or
+  (`name || 'you'`). What happens when the input is `null`, what happens when
+  the input is `''`, what happens when the input is ... `0`?
+- The difference between `===` and `==`, `!==` and `!=` ([equality comparisons and sameness][ref-equality]).
+  You should suggest they always use strict equality, but since the optimal
+  solution doesn't have a conditional, it does not apply to this exercise.
```

Comment:
If it does not apply to this exercise, is it even needed here? Can it be taken in further exercises?

Reply:
Yes, agreed.